docs/topics/tour/kotlin-tour-functions.md

```diff
@@ -387,7 +395,18 @@ The `.filter()` function accepts a lambda expression as a predicate:
 * `{ x -> x > 0 }` takes each element of the list and returns only those that are positive.
 * `{ x -> x < 0 }` takes each element of the list and returns only those that are negative.
 
-> If a lambda expression is the only function parameter, you can drop the function parentheses `()`.
+This example demonstrates two ways of passing a lambda expression to a function: 
```

Comment:
I like this, it shows very clearly how function arguments are not special at all (except for the trailing lambda syntax).